docs/pages/blog/base-ui-2024-plans.md

```diff
@@ -0,0 +1,105 @@
+---
+title: An exciting year ahead for Base UI
+description: The unstyled component library will get a stable release, lots of new components, and even better DX in 2024.
+date: 2024-02-09T00:00:00.000Z
+authors: ['michaldudak', 'danilo-leal']
+tags: ['Base UI', 'Product']
+card: true
+---
+
+The [story of Base UI](/blog/introducing-base-ui/) began several years ago—long before headless React component libraries skyrocketed in popularity—when we started to imagine a world in which Material UI could exist without Material Design.
+
+We're super excited to share that this dream has become a reality!
+This year will see a lot of investment in Base UI as we expand the team ([we're hiring!](/careers/staff-ui-engineer-base-ui/)) and focus hard on a [stable release](https://github.com/mui/material-ui/milestone/46) (tentatively planned for late 2024), which will come full of new components, features, and improvements.
+
+Let's walk through some of the things we're cooking up.
+
+## A larger set of components
+
+Base UI today offers a modest set of components and hooks, including some slightly more complex ones such as [Autocomplete](/base-ui/react-autocomplete/) and [Number Input](/base-ui/react-number-input/).
+However, we're aware that the package is still missing many primitive components that developers would need in order to adopt it for real-world applications.
+Fear not, because we're working hard to ship all of the components listed below with the stable release:
+
+| Components to be added |                                                                                                |
```

Comment:
Autocomplete/Combobox? We only have hooks

Reply:
Avatar, Scroll Area, Navigation Menu, Context Menu, at one point in the future would be great.

Reply:
Added a note about other components we might add in the future.

Reply:
> Avatar is controversial.

The value I see is a nice image-loading fallback mechanism: https://mui.com/blog/november-2019-update/